### PR TITLE
Add support for updating user's brand

### PR DESCRIPTION
**Pull request by nwithan8**
Finally, the C# library supports the ability to get update your brand with `user.UpdateBrand()`

This PR includes:
- The new `user.UpdateBrand()` function to update the associated user's brand. Pass in a `Dictionary<string, object>` as parameter. Function returns a `Brand` object.
- Docstring for this function.
- Added new `Brand` class, which features all necessary properties.
- A new `UserTest.TestUpdateBrand()` test for this feature

Classes, functions and tests have been imported/linked to all projects/versions, and all tests pass.

### CODE REVIEW

File: EasyPost.Tests/UserTest.cs [hunk 1]
```diff
@@ -18,6 +19,24 @@ public void TestRetrieveSelf()
             Assert.AreEqual(user.id, user2.id);
         }
 
+        [TestMethod]
+        public void TestUpdateBrand()
+        {
+            User user = User.Retrieve();
+            Assert.IsNotNull(user.id);
+
+            const string color = "#000000";
```
Comment (Justintime50) [inline EasyPost.Tests/UserTest.cs:28]: Let's use a color that's super random so we know this went through (AKA not a default white or black).

File: EasyPost.Tests/UserTest.cs [hunk 2]
```diff
@@ -18,6 +19,24 @@ public void TestRetrieveSelf()
...
+            Dictionary<string, object> details = new Dictionary<string, object>()
+            {
+                { "color", color }
+            };
+            Brand brand = user.UpdateBrand(parameters: details);
+            Assert.IsNotNull(brand);
+            Assert.IsNotNull(brand.user_id);
```
Comment (Justintime50) [inline EasyPost.Tests/UserTest.cs:35]: This second assertion is inherently useless as we already asserted the object isn't null - objects will always come back with IDs on them.

**Review by Justintime50 — APPROVED** [review]

File: EasyPost.Tests/UserTest.cs (hunk 2 above)
Comment (nwithan8) [reply EasyPost.Tests/UserTest.cs]: Resolved in [8d459a7](https://github.com/EasyPost/easypost-csharp/pull/203/commits/8d459a71ab9649ee007be4f7891e1ca9c06b371e)

File: EasyPost.Tests/UserTest.cs (hunk 1 above)
Comment (nwithan8) [reply EasyPost.Tests/UserTest.cs]: Resolved in [8d459a7](https://github.com/EasyPost/easypost-csharp/pull/203/commits/8d459a71ab9649ee007be4f7891e1ca9c06b371e)

**Review by Justintime50 — APPROVED** [review]

**Pull request merged**